File: add/codesnippet/CPP/4d10e835-c952-4b03-ba4f-_1.cpp
public:
   void FillRectanglesRectangle( PaintEventArgs^ e )
   {
      // Create solid brush.
      SolidBrush^ blueBrush = gcnew SolidBrush( Color::Blue );

      // Create array of rectangles.
      array<Rectangle>^ rects = {Rectangle(0,0,100,200),Rectangle(100,200,250,50),Rectangle(300,0,50,100)};

      // Fill rectangles to screen.
      e->Graphics->FillRectangles( blueBrush, rects );
   }

File: add/codesnippet/CSharp/4d10e835-c952-4b03-ba4f-_1.cs
        private void FillRectanglesRectangle(PaintEventArgs e)
        {

            // Create solid brush.
            SolidBrush blueBrush = new SolidBrush(Color.Blue);

            // Create array of rectangles.
            Rectangle[] rects = { new Rectangle(0, 0, 100, 200), new Rectangle(100, 200, 250, 50), new Rectangle(300, 0, 50, 100) };

            // Fill rectangles to screen.
            e.Graphics.FillRectangles(blueBrush, rects);
        }